migrations/20200113173707_add_shipment_type_to_mto_shipments.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TYPE mto_shipment_type AS ENUM (
+'HHG',
+'INTERNATIONAL HHG',
```

Comment:
Although we've been somewhat inconsistent so far, I think it would be better to avoid spaces in these values:

```suggestion
'INTERNATIONAL_HHG',
```